packages/core/src/render3/assert.ts

```diff
@@ -68,7 +68,7 @@ export function assertLView(value: any) {
   assertEqual(isLView(value), true, 'Expecting LView');
 }
 
-export function assertFirstTemplatePass(tView: TView, errMessage?: string) {
+export function assertfirstCreatePass(tView: TView, errMessage?: string) {
```

Comment:
Can we correct the camel case here?

```suggestion
export function assertFirstCreatePass(tView: TView, errMessage?: string) {
```